### ISSUE TITLE
consider getting rid of type aliases

### ISSUE THREAD

**Issue by snd** [paritytech/ethabi#98]
this is more a nitpick but something that's personally bugging me every time i use ethabi.

ethabi aliases a bunch of types:
https://github.com/paritytech/ethabi/blob/7b9cca375986b948d84530442d6bd27d480b24c8/ethabi/src/lib.rs#L56

among them `type Bytes = Vec<u8>;`
every time i come back to ethabi after a while i assume that `Bytes` is something special only to find that it's a plain and simple `Vec<u8>`. i'd like for me and others to get rid of this surprise.

similarly ` type Hash = ethereum_types::H256;`. using many crates that use `H256` i've become pretty accustomed to seeing/using it. seeing `Hash` breaks that familiarity. i find that it's unnecessary indirection.

i'd prefer to get rid of these indirections unless they add something crucial i'm missing.
i understand that using the aliases it's possible to exchange the actual types without having to do renames in consuming code. i'd argue renaming is the least work required (easily automated) in changing the actual type implementations.

as i said it's nitpicky and might just annoy me personally.
what are your thoughts?

**Comment by CPerezz** [comment]
Totally agree with @snd words. Bytes type is a good example to show this.
I've started using web3.rs crate, and when I've had to use some ethabi types to call or query contracts, it's a bit weird to find all of the redundancies that appear with types that are exactly the same but with different names.

In my opinion (which may not be as elaborated as yours because I've used Rust not that long) these things make the Web3 Rust implementation less attractive to new developers that are introducing themselves with this language and crates. 

It would be nice to standardize this Types to work always with the same.

**Comment by dvdplm** [comment]
Yeah I don't think the alias buys us much in terms or ergonmics. A PR to change this is welcome.
